package.json

```diff
@@ -155,7 +160,8 @@
     "rollup": "^2.38.5",
     "rollup-plugin-size-snapshot": "^0.12.0",
     "shx": "^0.3.3",
-    "typescript": "^4.1.3"
+    "typescript": "^4.1.3",
+    "xstate": "^4.16.0"
```

Comment:
`xstate` should be added as an optional peerDependency as well - without it, this won't work OK with strict package managers like yarn 2 & pnpm

Reply:
Oh, good point. I should do that for other ones too.